deploy/helm/sumologic/values.yaml

```diff
@@ -4342,6 +4342,22 @@ metadata:
               - batch
             exporters:
               - sumologic/containers
+          ## This is the same pipeline like for logs/fluent/containers with the following modifications:
+          ## - filter/include_fluent_tag_containers and attributes/remove_fluent_tag are being removed
+          ##   as only containers log are being provided to otlp receiver
+          ## - attributes/containers functionality is being replaced by otellogs operators
+          logs/otlp/containers:
+            receivers:
+              - otlp
+            processors:
+              - memory_limiter
+              - groupbyattrs/containers
+              - k8s_tagger
+              - source/containers
+              - resource/containers_copy_node_to_host
+              - batch
+            exporters:
+              - sumologic/containers
```

Comment:
Why are you uncommenting this?

Reply:
I believe there is no need to keep it commented. If customers do not use otellogs, they basically won't be sending any data

Reply:
and it will be easier for them to use it just by switching `otellogs.enabled`

Reply:
We originally commented it out because the `k8s_tagger` increases memory usage for no benefit if `otellogs` are disabled. I assumed we'd uncomment it when officially moving it out of experimental status in v3.

Reply:
```suggestion
          ## Uncomment this only if you're enabling the Otelcol Log Collector via otellogs.enabled
          ## This is the same pipeline like for logs/fluent/containers with the following modifications:
          ## - filter/include_fluent_tag_containers and attributes/remove_fluent_tag are being removed
          ##   as only containers log are being provided to otlp receiver
          ## - attributes/containers functionality is being replaced by otellogs operators
          # logs/otlp/containers:
          #   receivers:
          #     - otlp
          #   processors:
          #     - memory_limiter
          #     - groupbyattrs/containers
          #     - k8s_tagger
          #     - source/containers
          #     - resource/containers_copy_node_to_host
          #     - batch
          #   exporters:
          #     - sumologic/containers
```